teaspoons-cli/teaspoons/auth_helper.py

```diff
@@ -42,20 +52,25 @@ def _validate_token(token: str) -> bool:
         # This is just to ensure the token is not expired
         jwt.decode(token, options={"verify_signature": False, "verify_exp": True})
         return True
+    except jwt.ExpiredSignatureError:
+        LOGGER.info("Token expired")
+        return False
     except Exception as e:
+        LOGGER.error(f"Error validating token: {e}")
         return False
 
 
 def _clear_local_token(token_file: str):
     try:
         os.remove(token_file)
     except FileNotFoundError:
+        LOGGER.debug("No local token found to clean up")
```

Comment:
what level of logging do we plan to have for the CLI when published, ERROR + INFO?

Reply:
yeah INFO and up is the current setting. a DEBUG log will be picked up in tests. we don't yet have a way to run the cli directly with debug logging, can add that easily though